internal/pkg/consensus/block_validation.go

```diff
@@ -77,13 +97,6 @@ func (dv *DefaultBlockValidator) ValidateSemantic(ctx context.Context, child *bl
 		return fmt.Errorf("block %s has invalid height %d", child.Cid().String(), child.Height)
 	}
 
-	// check that child is appropriately delayed from its parents including
-	// null blocks.
-	// TODO replace check on height when #2222 lands
-	limit := uint64(pmin) + uint64(dv.BlockTime().Seconds())*(uint64(child.Height)-ph)
```

Comment:
Note: I've moved the epoch alignment check to "syntax" because the clock synchrony assumption means we no longer need the parent block for this.